packages/discovery-react-components/src/utils/document/processDoc.ts

```diff
@@ -302,6 +310,11 @@ function setupSectionParser(
         currentTable = null;
       }
 
+      if (doc.bboxes && tagName === BBOX_TAG && currentBbox) {
+        currentBbox.location.end = getChildEndFromCloseTag(p);
```

Comment:
Can you verify that `getChildEndFromCloseTag(p)` is correct, instead of `p.endIndex || p.startIndex` as in line 309 above?

Reply:
I did, My first try was to use `p.endIndex || p.startIndex` it did not return the right value.